lib/minfraud/assessments.rb

```diff
@@ -129,18 +136,22 @@ def score
     private
 
     def perform_request(endpoint)
-      raw = request.perform(
-        verb:     :post,
-        endpoint: endpoint.to_s,
-        body:     request_body,
-      )
+      response = nil
+      body     = nil
+      Minfraud.connection_pool.with do |client|
+        response = client.post(
+          "/minfraud/v2.0/#{endpoint}",
+          body: JSON.generate(request_body),
```

Comment:
Is this better than `:json" => request_body` or whatever? I think that would also set the correct content type, which we don't seem to be doing currently.

Reply:
It'd be basically the same other than the header would be set I think. I'll switch it!